src/ngx_http_lua_timer.c

```diff
@@ -25,7 +25,8 @@ typedef struct {
      * ngx_connection_t. and we use the Lua coroutine reference number as
      * the event ident */
     int           co_ref;
-    unsigned      premature;  /* :1 */
+    unsigned      premature:1;
+    unsigned      delay:31;
```

Comment:
It's better to put larger fields *before* smaller fields in a struct. Can we move them to the end and reorder their order?